packages/core/src/lib/internal/net/FetchTransport.ts

```diff
@@ -21,13 +22,9 @@ export default class FetchTransport implements Transport {
       const body = values[1];
       const headers: { [key: string]: string } = {};
       response.headers.forEach((key, value) => {
-        headers[key] = value;
+        headers[value] = key;
```

Comment:
[nit] even if this is right, this appears incorrect. If the headers in the fetch response are actually reversed, change the foreach to be 
```
response.headers.forEach((value, key) => {
    headers[key] = value
```